Assets/Scripts/TakeDamageFromPlayerProjectile.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+using System.Linq;
+
+public class TakeDamageFromPlayerProjectile : MonoBehaviour
+{
+    public float health = 25.0f;
+
+    void OnCollisionEnter(Collision collision)
+    {
+        if (collision.gameObject.CompareTag(Tags.PLAYER_PROJECTILE))
+        {
+            float force = collision.relativeVelocity.magnitude;
+            health -= force;
```

Comment:
nit: rename this variable to damage instead. It reads a bit weirdly.

```suggestion
            float damage = collision.relativeVelocity.magnitude;
            health -= damage;
```